yt/yt/server/node/exec_node/slot_manager.h

```diff
@@ -228,7 +228,7 @@ class TSlotManager
     EJobEnvironmentType JobEnvironmentType_;
 
     //! We maintain queue for distributing job logs evenly among slots.
-    TRingQueue<int> FreeSlots_;
+    TRingQueue<int> FreeSlots_;  // For usage in main thread only, use bootstrap job invoker to run via
```

Comment:
We write comments at previous line.
If you want to write explicit, comment, you can just write
// Invoker affinity: JobInvoker